xml/docu_styleguide.content.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="sec-content">
+ <title>Documentation content</title>
+
+ <para>
+  When selecting what to document, keep to the following guidelines:
+ </para>
+
+ <itemizedlist>
+  <listitem>
+   <para>
+    Do not promise future features.
+    Only document features that exist already or that will be finished before
+    the document is first published.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    In some cases, it is appropriate to warn of scheduled future changes,
+    such as feature removals.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Documentation for unsupported features should be an exception.
```

Comment:
```suggestion
    Documentation concerning unsupported features should be an exception.
```

Reply:
What's the advantage of the change here?

Reply:
"Documentation for X" implies that the purpose of the documentation, i.e. its intended audience, is X. 

"Docs _about_ X" or "docs _on_ X" or "docs _of_ X" are all preferable in some ways, but all are potentially ambiguous. So after trying a few alternatives, "concerning" seems the least ambiguous.

Reply:
Linguee may not always be right, but it does not seem to back up the idea that "documentation for" implies an audience rather than a feature/product: https://www.linguee.de/deutsch-englisch/search?source=auto&query=documentation%20for ,
Stack Exchange seem to support the idea either: https://ell.stackexchange.com/questions/3292/#3297

Reply:
Ok, I took it anyway, but in case that for/concerning rule is indeed a general rule, we probably have a huge issue here with much of our documentation.